docs/index.mdx

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
In case we don't need an index file at the root level for technical reasons we might as well remove this?